docs/components/board/upboard.md

```diff
@@ -21,7 +22,7 @@ Follow [these instructions](https://ubuntu.com/tutorials/install-ubuntu-server)
 Follow [these instructions](https://github.com/up-division/pinctrl-upboard) to do so.
 This driver stabilizes the [GPIO pin mapping definition](https://github.com/up-board/up-community/wiki/Pinout) on the board to make it identical to that of a [Raspberry Pi](/components/board/pi/).
 
-{{< tabs name="Configure an upboard Board" >}}
+<!-- {{< tabs name="Configure an upboard Board" >}}
 {{% tab name="Config Builder" %}}
```

Comment:
Commented out as it's not actually visible in the app, so you can't use the config builder for it. At least it isn't for me right now.